front/src/config/i18n/fr.json

```diff
@@ -1472,6 +1473,14 @@
     "invalidIcon": "L'icône est requise",
     "sceneAlreadyExist": "Une scène avec le même nom existe déjà."
   },
+  "duplicateScene": {
+    "cardTitle": "Copier une scène",
+    "nameLabel": "Nom",
+    "namePlaceholder": "Entrez le nom de votre scène",
+    "duplicateSceneButton": "Copier une scène",
```

Comment:
The button could be different from the title, it's weird to have a generic button label here.

Usually we just put the action, like here "Dupliquer".